src/scheduler/data_types.h

```diff
@@ -372,6 +372,7 @@ struct queue_info
 	unsigned has_soft_limit:1;	/* queue has a soft user/grp limit set */
 	unsigned has_hard_limit:1;	/* queue has a hard user/grp limit set */
 	unsigned is_peer_queue:1;	/* queue is a peer queue */
+	unsigned has_res_av:1;		/* queue has resources_available limits */
```

Comment:
I think we should add the word 'limit' to the identifier, maybe 'has_resav_limit' ?

Reply:
+1 on changing the name to `has_res_av_limit`.

Reply:
I changed the name.